src/lib.rs

```diff
@@ -134,6 +131,24 @@ pub struct Rec<P>(PhantomData<P>);
 /// out of.
 pub struct Var<N>(PhantomData<N>);
 
+trait HasReceiver {
```

Comment:
Why is this necessary? To allow pulling the `Receiver<_>` out of different channels types?

EDIT Doesn't appear to be used.